tests/publiccloud/nvidia.pm

```diff
@@ -0,0 +1,63 @@
+# SUSE's openQA tests
+#
+# Copyright @ SUSE LLC
+# SPDX-License-Identifier: FSFAP
+
+# Summary: Opensource Nvidia test
+#          Drivers are only supported on SLE15SP4+
+# Maintainer: ybonatakis <ybonatakis@suse.com>
+
+use Mojo::Base 'consoletest';
```

Comment:
```suggestion
use Mojo::Base 'publiccloud::basetest';
```

this way you will get auto-cleanup logic

Reply:
*ready for review*